OneWelcomeExampleApp/Interactors/WelcomeInteractor.swift

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright © 2022 OneWelcome. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+protocol WelcomeInteractorProtocol: AnyObject {
+    func handleInterfaceStyleChange(_ interfaceStyle: UIUserInterfaceStyle)
```

Comment:
the class cannot built without an import to UIKit, but...

the interactor should not depend on it, thus you need to proxy the type here and pass the raw value from VC